pkg/lib/deps/deps_provider.go

```diff
@@ -88,6 +90,7 @@ var AppRootDeps = wire.NewSet(
 	wire.Bind(new(event.Database), new(*appdb.Handle)),
 	wire.Bind(new(workflow.ServiceDatabase), new(*appdb.Handle)),
 	wire.Bind(new(authenticationflow.ServiceDatabase), new(*appdb.Handle)),
+	wire.Bind(new(handlerwebappauthflowv2.ResetPasswordHandlerDatabase), new(*appdb.Handle)),
```

Comment:
Try to move this to auth/deps.go to avoid the lint error?

Reply:
Thanks, it worked 🙇 curious - does it matter (in terms of code structure) where we declare these wire DI?

If not - why not just put all wire bindings in one holy file? Since not these `deps` file are too big to be readable anyway

Reply:
> for maintaining a certain import structure

Oh, thanksss 🙏